components/PokemonListRow.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { StyleSheet, Text } from 'react-native';
+import { colors } from '../style/styleVariables';
+
+interface Props {
+  pokemonId: string;
+  pokemonName: string;
+  onPokemonListRowClicked: (pokemonName: string) => void;
+}
+
+function propsAreEqual(prevProps: Props, nextProps: Props) {
```

Comment:
it's fine, but what about `onPokemonListRowClicked`? -> it may also change (if `navigation` object would change) -> there's small chance for that, but I'm not sure if there's a need to write custom `propsAreEqual` method. Simple `React.memo(PokemonListRow)` should be fine I think